core-services/release-controller/_releases/release-ocp-4.19.json

```diff
@@ -505,7 +505,7 @@
     "qe-perfscale": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes"
+        "name": "periodic-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes"
```

Comment:
Where is the actual job named: `periodic-ci-openshift-eng-ocp-qe-perfscale-ci-main-aws-4.19-nightly-x86-payload-control-plane-6nodes` defined?  
The release-controller config failures you're getting are directly related to this question

Reply:
talked with Brad offline and missed updating [this file](https://github.com/openshift/release/blob/129e601a900964133a1f47501f2fdbc433a018fe/hack/validate-release-jobs.py) where it defines which job files are used in release controller